action/sealedenvelope.go

```diff
@@ -130,6 +123,11 @@ func (sealed *SealedEnvelope) Encoding() uint32 {
 	return uint32(sealed.encoding)
 }
 
+// ToEthTx converts to Ethereum tx
+func (sealed *SealedEnvelope) ToEthTx() (*types.Transaction, error) {
+	return sealed.Envelope.ToEthTx(sealed.evmNetworkID, sealed.encoding)
```

Comment:
converting to EVM tx requires knowing the evmNetworkID, so it is natural `ToEthTx()` should be a func of `SealedEnvelope`